src/chains/strategies/bsc.strategy.ts

```diff
@@ -1,7 +1,11 @@
 import { BadRequest, UnprocessableEntity } from '@tsed/exceptions';
 import { ethers } from 'ethers';
 import { TOKENS } from '../../config/constants';
+<<<<<<< HEAD
 import { getTokenPrice, getVaultTokenPrice, getWrapperTokenPrice } from '../../prices/prices-util';
+=======
+import { getTokenPrice } from '../../prices/prices-util';
+>>>>>>> 2e7d652 (feat: initial v2 working implementation (#52))
```

Comment:
Fork is a great git client. =)

Reply:
yeah might look into it. this particular one had like probaly 40 conflicts, so i missed a few as i went through by folder